setupTests.ts

```diff
@@ -8,3 +11,11 @@ export const buildRulesetExceptionCollectionFrom = (
   source[loc] = rules;
   return source;
 };
+
+export const normalizeSeverityFromJsonRuleset = (severity: string | number | undefined): DiagnosticSeverity => {
```

Comment:
I believe we have a very similar function somewhere.
Most likely in merger, suggest consolidating.

Reply:
@P0lip I'm going to table this until #974 is merged. The way the oas3 tests are written make this change especially painful... Too painful.

Either I'm approaching this from the wrong angle or some refactoring should be done.

🤔